docker/Dockerfile

```diff
@@ -52,11 +52,19 @@ RUN apt-get update &&apt-get install -y \
     libsgx-ra-uefi
 
 
-# Build App from source
+#  Install jsoncpp && Build App from source
 WORKDIR /home
-RUN git clone https://github.com/intel/ehsm.git \
-   && cd ehsm \
-   && make
+RUN git clone https://github.com/microsoft/vcpkg.git \
+    && git clone -b enroll_M0 https://github.com/Liang1XZhao/ehsm.git \
+    && cd vcpkg \
+    && ./bootstrap-vcpkg.sh \
+    && ./vcpkg integrate install \
+    && ./vcpkg install jsoncpp \
+    && cp ./installed/x64-linux/lib/libjsoncpp.a /usr/local/lib \
+    && cp -r ./installed/x64-linux/include/json /home/ehsm/include \
+    && cp -r /usr/include/x86_64-linux-gnu/curl /home/ehsm/include \
```

Comment:
can the below work?
apt install libjsoncpp-dev, libcurl4-openssl-dev or libcurl4-gnutls-dev

Reply:
Can you try this:
1. apt install libjsoncpp-dev
2. add #include <jsoncpp/json/json.h> to your file
3. g++ compile

Reply:
I will try it